js/src/index.ts

```diff
@@ -159,6 +322,24 @@ export class Schematic {
     context: SchematicContext,
     fallback: boolean,
   ): Promise<boolean> {
+    // If in offline mode, immediately return fallback value
+    if (this.isOffline()) {
+      this.debug(`fallbackToRest offline result: ${key}`, {
+        value: fallback,
+        offlineMode: true,
+      });
+
+      // Create a minimal result for the offline case and submit event
+      const offlineResult: CheckFlagReturn = {
+        flag: key,
+        value: fallback,
+        reason: "Offline mode enabled (fallback)",
+      };
+      this.submitFlagCheckEvent(key, offlineResult, context);
```

Comment:
hmm, should we submit on offline? I feel like no, but I could see this going either way. 

I feel like if ppl want offline to reduce some network activity, then no events is the cost -- it is _offline_ for a reason

Reply:
no we should remove this. it won't even submit anyway, it wil just log that it would have submitted an event